.github/workflows/maven.yaml

```diff
@@ -118,26 +113,63 @@ jobs:
           CURRENT_INDEX: ${{ matrix.current_index }}
           NUMBER_OF_JOBS: ${{ matrix.number_of_jobs }}
         run: |
-          # find all tests
-          # exclude Fabric8IstionIT
-          # only take classes that have @Test inside them
-          # drop the "begining" xxx/src/test/java
-          # replace / with .
-          # drop last ".java"
-          # replace newline with space
-          # sort
-
-          CLASSNAMES=($(find . -name '*.java' \
-          				 | grep  'src/test/java' \
-          				 | grep -v 'Fabric8IstioIT' \
-          				 | xargs grep -l '@Test' \
-          				 | sed 's/.*src.test.java.//g' \
-          				 | sed 's@/@.@g' \
-          				 | sed 's/.\{5\}$//' \
-                         | sort \
-          				 | tr '\n' ' '))
-
-          number_of_tests=${#CLASSNAMES[@]}
+          # - find all tests
+          # - exclude Fabric8IstionIT
+          # - only take classes that have @Test inside them
+          # - ignore the ones that have 'abstract class'. we do this because otherwise we would pass
```

Comment:
this small comments explain the problem, I noticed this by accident and could reproduce locally too... thought appropriate to fix it in this PR